ddp-workspace/projects/ddp-sdk/src/lib/services/activity/activityQuestionConverter.service.ts

```diff
@@ -232,4 +237,9 @@ export class ActivityQuestionConverter {
             .filter(block => block !== null);
         return newBlock;
     }
+
+    private getFileBlock(questionJson: any): ActivityFileQuestionBlock {
+        const fileBlock = new ActivityFileQuestionBlock();
+        return fileBlock;
```

Comment:
#minor  `return  new ActivityFileQuestionBlock();`

Reply:
It is kind of `sketch`. 
Likely, I will add some specific properties for the `fileBlock` as it done for other blocks.